Jenkinsfile

```
@@ -1,5 +1,5 @@
-buildPlugin(useAci: false, configurations: [
-  [ platform: "linux", jdk: "8", jenkins: null ],
-  [ platform: "windows", jdk: "8", jenkins: null ],
-  [ platform: "linux", jdk: "11", jenkins: "2.276", javaLevel: 8 ]
+buildPlugin(useContainerAgent: false, configurations: [
```

Or
```suggestion
buildPlugin(useContainerAgent: true, configurations: [
```
if https://github.com/jenkinsci/workflow-cps-plugin/pull/355#issuecomment-624779205 has been cleared up.

---

Going to give it a shot...